apps/graph-worker/src/graph_publisher/graph.publisher.processor.service.ts

```diff
@@ -118,6 +119,9 @@ export class GraphUpdatePublisherService extends BaseConsumer implements OnAppli
 
       this.logger.debug('Cached extrinsic to monitor: ', payWithCapacityTxHash);
     } catch (error: unknown) {
+      if (error instanceof DelayedError) {
```

Comment:
`process()` is not calling `capacityCheckerService` above (like the other services), should it be?

Reply:
no the payWithCapacity and payWithCapacityBatchAll functions internally do that check do consumers dont have to worry about it

Reply:
this is mostly checking capacity only, but can be used else though the current implementation wants to make sure a tx is only sent if there is enough capacity